File: app/src/main/java/com/ashlikun/core/simple/MainFragment.java
package com.ashlikun.core.simple;

import android.util.Log;

import com.ashlikun.core.fragment.BaseMvpFragment;

public class MainFragment extends BaseMvpFragment<MainFragmentPresenter>
        implements IMainView {


    @Override
    public int getLayoutId() {
        Log.e("MainFragment", "getLayoutId");
        return R.layout.activity_main;

    }

    @Override
    public void initView() {
        Log.e("MainFragment", "initView");
    }


    @Override
    public void clearData() {
    }
}

File: app/src/main/java/com/ashlikun/core/simple/MainFragment.kt
package com.ashlikun.core.simple

import android.util.Log

import com.ashlikun.core.fragment.BaseMvpFragment

class MainFragment : BaseMvpFragment<MainFragmentPresenter>(), IMainView {


    override fun getLayoutId(): Int {
        Log.e("MainFragment", "getLayoutId")
        return R.layout.activity_main

    }

    override fun initView() {
        Log.e("MainFragment", "initView")
    }


    override fun clearData() {

    }

    override fun findSize() = 111
}
